docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -58,16 +91,51 @@ To follow this guide, you need:
    local-tempo-1        "/tempo -config.file…"   tempo               running             0.0.0.0:3200->3200/tcp, 0.0.0.0:4317-4318->4317-4318/tcp, 0.0.0.0:9411->9411/tcp, 0.0.0.0:14268->14268/tcp
    ```
 
-1. As part of the docker compose manifest, Grafana is now accessible on port 3000. Open a browser to localhost:3000 and sign in to Grafana.
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Explore the traces in Grafana
+
+As part of the docker compose manifest, Grafana is now accessible on port 3000. We can use Grafana to explore the traces generated by the k6-tracing service.
+
+1. Open a browser and navigate to [http://localhost:3000](http://localhost:3000).
 
 1. Once logged in, navigate to the **Explore** page, select the **Tempo** data source and select the **Search** tab. Select **Run query** to list the recent traces stored in Tempo. Select one to view the trace diagram:
-    <p align="center"><img src="../../setup/assets/setup-linux-run-query.png" alt="Query example"></p>
+   
+   {{< figure align="center" src="../../setup/assets/setup-linux-run-query.png" alt="Query example" >}}
+    
 
-1. A couple of minutes after Tempo has started up successfully, select the **Service graph** tab for the Tempo data source in the **Explore** page. Select **Run query** to view a service graph, generated by Tempo’s metrics-generator.
-    <p align="center"><img src="../../setup/assets/setup-linux-node-graph.png" alt="Service graph sample"></p>
+2. A couple of minutes after Tempo has started up successfully, select the **Service graph** tab for the Tempo data source in the **Explore** page. Select **Run query** to view a service graph, generated by Tempo’s metrics-generator.
 
-1. To stop the services:
-    ```
+   {{< figure align="center" src="../../setup/assets/setup-linux-node-graph.png" alt="Service graph sample" >}}
+
+3. To stop the services:
+    ```bash
     docker compose down -v
     ```
     
+<!-- INTERACTIVE page step2.md END -->
+
+<!-- INTERACTIVE page finish.md START -->
+
+# Next steps
```

Comment:
This is an automated thing from the validator. I'm not sure how to escape around this. 

@jdbaldry Have you dealt with this before?

Reply:
Ooh this is a good one. This is usually a correct positive because a page shouldn't have two H1 headings but it makes sense use a H1 here because it's the first heading in that course's page.

I think `doc-validator` needs a comment based mechanism for ignoring parts of the code and I may be able to find some time to work on that today.

In the short term, you can make this section start with an H2 and know that the transformer tool will make it an H1: https://github.com/grafana/killercoda/blob/staging/tools/transformer/transform.go#L233-L262

Reply:
Hi all, sorry that was completely my mistake. It should have been a H2 so now made that fix!